lib/cmd/license-builder.js

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs');
+const exec = require('child_process').exec;
+const getLicenseCommand = 'license-checker --production --json';
+
+class LicenseBuilder {
```

Comment:
The correct name from this class shouldn't be ThirdPartyLicenseGenerator or ThirdPartyLicenseWriter?